src/strings/PowerFxResources.en-US.resx

```diff
@@ -5158,4 +5158,33 @@
     <value>Can't delegate {0}: contains a behavior function '{1}'.</value>
     <comment>Warning message.</comment>
   </data>
+  <data name="AboutIsMatch" xml:space="preserve">
+    <value>IsMatch is used to validate a string corresponds to a defined pattern.</value>
```

Comment:
>IsMatch is used to validate a string corresponds to a defined pattern. [](http://example.com/codeflow?start=11&length=70)

Should we have same description as shown in PA?
(Would be great if we could just have common implementation moved over from PA)

Reply:
Yes